downloader_library/src/main/java/com/google/android/vending/expansion/downloader/impl/DownloadNotification.java

```diff
@@ -130,22 +176,28 @@ public void onDownloadStateChanged(int newState) {
                     ongoingEvent = true;
                     break;
             }
+
             mCurrentText = mContext.getString(stringDownloadID);
-            mCurrentTitle = mLabel.toString();
-            mCurrentNotification.tickerText = mLabel + ": " + mCurrentText;
-            mCurrentNotification.icon = iconResource;
-            mCurrentNotification.setLatestEventInfo(mContext, mCurrentTitle, mCurrentText,
-                    mContentIntent);
+            mCurrentTitle = mLabel;
+            mCurrentBuilder.setTicker(mLabel + ": " + mCurrentText);
+            mCurrentBuilder.setSmallIcon(iconResource);
+            mCurrentBuilder.setContentTitle(mCurrentTitle);
+            mCurrentBuilder.setContentText(mCurrentText);
             if (ongoingEvent) {
-                mCurrentNotification.flags |= Notification.FLAG_ONGOING_EVENT;
+                mCurrentBuilder.setOngoing(true);
             } else {
-                mCurrentNotification.flags &= ~Notification.FLAG_ONGOING_EVENT;
-                mCurrentNotification.flags |= Notification.FLAG_AUTO_CANCEL;
+                mCurrentBuilder.setOngoing(false);
+                mCurrentBuilder.setAutoCancel(true);
+            }
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                notificationManager.notify(NOTIFICATION_ID, mCurrentBuilder.build());
+            } else {
+                mNotificationManager.notify(NOTIFICATION_ID, mCurrentBuilder.build());
```

Comment:
Check this _if-else_

Reply:
Yes this is required to make the notification compatible across Android versions